prdoc/pr_7286.prdoc

```diff
@@ -0,0 +1,20 @@
+title: Remove feature flag checks for Elastic Scaling MVP
+doc:
+- audience: Runtime Dev
```

Comment:
All of these changes I can see in this PR are on the node side, but as @alindima pointed out, let's also cleanup the runtime.